src/utilities/graph-util.js

```diff
@@ -175,6 +175,56 @@ class GraphUtils {
   static isEqual(prevNode: any, newNode: any) {
     return fastDeepEqual(prevNode, newNode);
   }
+
+  static findNodesWithinArea(start: IPoint, end: IPoint, nodes: INode[]) {
+    const smallerX = start.x <= end.x ? start.x : end.x;
```

Comment:
Nit: Use Math.min() and Math.max() instead of ternary operator

Reply:
Thanks. Forgot about that.